### PR TITLE
search tests added

### PR DESCRIPTION
**Pull request by t-karasova**

### CODE REVIEW

File: search/search_with_filtering.py [hunk 1]
```diff
@@ -30,14 +30,14 @@ def get_search_service_client():
...
-def get_search_request(query: str, _filter: str, page_size=10):
+def get_search_request(query: str, _filter: str):
     default_search_placement = "projects/" + project_number + "/locations/global/catalogs/default_catalog/placements/default_search"
 
     search_request = SearchRequest()
     search_request.placement = default_search_placement  # Placement is used to identify the Serving Config name.
     search_request.query = query
     search_request.filter = _filter
-    search_request.page_size = page_size
+    search_request.page_size = 10
```
Comment (pmichalskiGD) [inline search/search_with_filtering.py:40]: are you hard coding values to match tests?

File: search/update_attribute_configuration.py [hunk 2]
```diff
@@ -72,8 +72,9 @@ def update_product(product_to_update_id):
     print(updated_product)
 
     print('---Wait 5 minutes to be sure the catalog has been indexed after the changes---:')
-    time.sleep(300)
+    time.sleep(60)
```
Comment (pmichalskiGD) [inline search/update_attribute_configuration.py:75]: print says you wait 5 minutes, but sleep is for 1 min

**Review by pmichalskiGD — COMMENTED** [review]
product/create_gcs_bucket.py  is empty
plenty of pyc files have been added. Is this intended?

File: search/search_with_filtering.py (hunk 1 above)
Comment (t-karasova) [reply search/search_with_filtering.py:40]: no, just to limit the products in the output, 10 products are fully displayed in the Terminal

File: search/update_attribute_configuration.py (hunk 2 above)
Comment (t-karasova) [reply search/update_attribute_configuration.py:75]: changed to 2 minutes

**Pull request merged**